tests/unit/customizations/eks/test_get_token.py

```diff
@@ -103,3 +114,33 @@ def test_token_sess(self):
             token_no_prefix.encode()
         ).decode()
         self.assert_url_correct(decrypted_token, True)
+
+class TestGetTokenCommand(unittest.TestCase):
+
+    @patch('awscli.customizations.eks.get_token.uni_print')
```

Comment:
You should use `testutils.capture_output` rather than patching `uni_print` directly.

Reply:
Done